src/nl/hannahsten/texifyidea/inspections/latex/probablebugs/LatexQuoteInspection.kt

```diff
@@ -0,0 +1,27 @@
+package nl.hannahsten.texifyidea.inspections.latex.probablebugs
+
+import com.intellij.codeInspection.ProblemHighlightType
+import nl.hannahsten.texifyidea.inspections.TexifyRegexInspection
+import java.util.regex.Pattern
+
+/**
+ * @author Michael Milton
+ */
+class LatexQuoteInspection : TexifyRegexInspection(
+        inspectionDisplayName = "Incorrect quotation",
+        inspectionId = "AsciiQuotes",
+        errorMessage = { """"ASCII quotes" were used instead of ``LaTeX quotes''""" },
+        highlight = ProblemHighlightType.GENERIC_ERROR_OR_WARNING,
+        pattern = Pattern.compile("""".+?"|'.+?'"""),
```

Comment:
I found some false positives to consider:

```latex
\documentclass{article}
\usepackage{csquotes}
\MakeOuterQuote{"}
\begin{document}
Think about `procedure', `second', day and also atmospheres. % False positive
Specifying of `algorithms' flows this length of 2'\thinspace3''. % False positive
Storm is "done" cooperative. % False positive because csquotes
This is “carefully” “blue”. % True negative
\end{document}
```

Also, 
```
``quote"
```
 is typeset the same as 
```
``quote''
```
and I couldn't find any definite source saying that the " should not be used as right quotes. But it doesn't look very nice to me, so I would agree with discouraging this. Though in that case we should ensure that the quickfix for
```
The algorithm ``appends" ``everything" related to the specificality of surveys.
```
would produce the correct result, so perhaps it would be easier to filter out this case.

Please add tests for any case that you fix.